core/src/css/component/replays/replay-info/replay-info.component.scss

```diff
@@ -108,9 +108,9 @@
 
 	&.time {
 		font-size: 13px; 
-		width: 70px;
+		width: 80px;
 		padding: 0;
-		justify-content: flex-end; 
+		justify-content: flex-start;
```

Comment:
Well, this change is the whole point. This is necessary so that time doesn't overlap with separator.
`width: 80px;` is necessary to avoid overlapping from the other side with a small window size.

Reply:
I don't understand why the justify-content change prevents the overlap. It should only dictate whether the content is aligned at the start or at the end (and I would like to have it aligned at the end).

Isn't the width change enough?

Reply:
Without `justify-content: flex-start;` it looks like this. 

![2022-09-05_22-46-56](https://user-images.githubusercontent.com/43519401/188507048-129ff3b2-fefc-4bcd-8bc3-6306decf1f6d.png)

Reply:
I see what you mean. However all times are aligned, which makes it easier to see the longer games IMO. I'm wondering if there could be a solution to have the best of both worlds. I'll think about it.

Reply:
I have already done this (see below).